src/frontend/shortlist/src/components/layout/shortlists/ShortlistSchoolIcon.vue

```diff
@@ -10,10 +10,18 @@ export default {
   >
     <img class="school-icon-img" :src="schoolData.img" />
     <div class="school-icon-name shortlist-fnt-serif-bold">
+      <!-- Old school data format -->
+      <div v-if="schoolData.name">
       {{ schoolData.name }}
+      </div>
+      <!-- New school data format -->
+      <div v-else>
+        {{ schoolData.school.schoolMetadata.name }}
+      </div>
```

Comment:
Could you please explain why the first if uses data from schoolData, and the else uses data from schoolMetadata?

Reply:
The old format is for the dummy API in shortlists.js (note plural) while the new format is for the real API data. Since both are being used this is just so they can both be displayed.